libbeat/beat/beat.go

```diff
@@ -52,15 +52,7 @@ import (
 	"github.com/satori/go.uuid"
 )
 
-var (
-	printVersion = flag.Bool("version", false, "Print the version and exit")
-)
-
-var debugf = logp.MakeDebug("beat")
-
-// GracefulExit is an error that signals to exit with a code of 0.
-var GracefulExit = errors.New("graceful exit")
-
+/*
 // Beater is the interface that must be implemented every Beat. The full
```

Comment:
yes, just commented out original interface. still needs to be cleaned up here. `in progress` flag ;)